blog/articles/get-started-knative-eventing.md

```diff
@@ -0,0 +1,93 @@
+| Title  | LinkTitle | Date | Description | Type | Aliases
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+| Getting started with Knative Eventing using Bitcoin transaction data  | Get Started with Knative Eventing  | 2020-05-01  | Guided walkthrough on using Knative Eventing  | Blog  | /blog/2010-05-01-getting-started-knative-evetning  |
+
+I’ve been exploring [Knative Eventing](https://knative.dev/docs/eventing/), a system that enables a cloud native eventing ecosystem to be easily deployed through the use of **event producers** and **event consumers.**
+
+As part of this I put together a simple demo that someone can use to explore, what I think are, some of the key advantages of event-driven architectures, such as:
+
+- Push-based messaging.
+- True decoupling of producers and consumers.
+- Apply business logic while data is in motion.
+- Real time event-streams for data science — millisecond decisions e.g fraud detection.
+
+In this post, I’ll show you how to get an example eventing scenario up and running using some of the basic Knative eventing components; **broker, trigger, producer** and **consumer.** This demo shows streaming events in real-time, in-stream transformation and push-based front-ends in action and they can serve as a basis for you to explore further. You can use this as a basis and build out, exploring further with what is possible.
```

Comment:
I think we should capitalize Broker and Trigger, since they are actual resources in Knative.